AreaShop/src/main/java/me/wiefferink/areashop/commands/CommandAreaShop.java

```diff
@@ -81,11 +86,44 @@ public boolean confirm(CommandSender sender, String[] args, Message message) {
 			return true;
 		}
 
-		message.prefix().append(Message.fromKey("confirm-yes").replacements(command)).send(sender);
+		Message m = message.prefix().append(Message.fromKey("confirm-yes").replacements(command));
+		send(m, sender);
 		lastUsed.put(sender.getName(), new CommandTime(command, now));
 		return false;
 	}
 
+	private void send(Message message, CommandSender target) {
+		if(AreaShop.useMiniMessage())
+		{
+			if(message.get() == null || message.get().size() == 0 || (message.get().size() == 1 && message.get().get(0).length() == 0) || target == null) {
+				return;
+			}
+			message.doReplacements();
+
+			StringBuilder messageStr = new StringBuilder();
+			for(String line : message.get())
+			{
+				messageStr.append(line);
+			}
+
+			MiniMessage mm = MiniMessage.miniMessage();
+			TextComponent parsed = (TextComponent) mm.deserialize(messageStr.toString());
+			try
+			{
+				Audience audience = (Audience) target;
+				audience.sendMessage(parsed);
+			}
+			catch (ClassCastException e)
+			{
+				Bukkit.getLogger().severe("AreaShop sent a non-supported Object as the Audience for a Message!");
```

Comment:
Do not use the `Bukkit` logger under any circumstances. Either use `Plugin#getLogger` or `AreaShop#warn` or `AreaShop#error`